src/home/components/SignupContents/EmailForm/useEmailForm.ts

```diff
@@ -1,12 +1,14 @@
 import { useState } from 'react';
 
+import { isEmail } from '@yourssu/utils';
+
 import { postAuthVerificationEmail } from '@/home/apis/authVerification.ts';
 import { EmailFormProps } from '@/home/components/SignupContents/EmailForm/EmailForm.type.ts';
 import { useParseFullEmail } from '@/hooks/useParseFullEmail';
 
 export const useEmailForm = ({ onConfirm }: EmailFormProps) => {
   const [email, setEmail] = useState('');
-  const [emailError, setEmailError] = useState<string | undefined>(undefined);
+  const [emailError, setEmailError] = useState<string | null>(null);
   const parseFullEmail = useParseFullEmail();
```

Comment:
```suggestion
  const [emailError, setEmailError] = useState<string>('');
```
`undefiend` 대신 빈 문자열을 기본값으로 넣어주면 어떨까요?

Reply:
[refactor: emailError가 없을 시 빈 문자열로 표시](https://github.com/yourssu/Soomsil-Web/pull/300/commits/aa1c7298b7d36ccc4730140f0301e33e476119d1)에서 빈 값으로 변경했습니다!